app/src/main/java/de/tum/in/tumcampusapp/activities/wizard/WizNavStartActivity.java

```diff
@@ -41,14 +47,54 @@ public void onCreate(Bundle savedInstanceState) {
 		LinearLayout layout = (LinearLayout) findViewById(R.id.wizard_start_layout);
 		layout.requestFocus();
 
+        userMajorSpinner=(Spinner)  findViewById(R.id.majorSpinner);
 		editText = (EditText) findViewById(R.id.lrd_id);
-
+        setUpSpinner();
 		lrzId = sharedPrefs.getString(Const.LRZ_ID, "");
 		if (lrzId != null) {
 			editText.setText(lrzId);
 		}
 	}
 
+    public void setUpSpinner()
+    {
+        String select = "Please Select Your Major";
+        String math = getResources().getString(R.string.faculty_mathematics);
+        String physics = getResources().getString(R.string.faculty_physics);
+        String chemistry = getResources().getString(R.string.faculty_chemistry);
+        String tum_manag = getResources().getString(R.string.faculty_tum_school_of_management);
+        String cge = getResources().getString(R.string.faculty_civil_geo_and_environmental_engineering);
+        String architecture = getResources().getString(R.string.faculty_architecture);
+        String mechanical = getResources().getString(R.string.faculty_mechanical_Engineering);
+        String electrical = getResources().getString(R.string.faculty_electrical_and_computer_engineering);
+        String informatics = getResources().getString(R.string.faculty_informatics);
+        String tum_life_sc = getResources().getString(R.string.faculty_tum_school_of_life_sciences_weihenstephan);
+        String medicine = getResources().getString(R.string.faculty_tum_school_of_medicine);
+        String sport = getResources().getString(R.string.faculty_sport_and_health_sciences);
+        String edu = getResources().getString(R.string.faculty_tum_school_of_education);
+        String political_social = getResources().getString(R.string.faculty_political_and_social_sciences);
+        String[] majors = {select,math, physics, chemistry, tum_manag, cge, architecture, mechanical, electrical, informatics, tum_life_sc, medicine, sport, edu, political_social};
```

Comment:
The majors selection is duplicated / copypasted code from the SurveyActivity. You shouldn't do that. Also I have a feeling, that you should get those majors from TUMOnline instead of hardcoding them